boilerplate/app.config.ts

```diff
@@ -0,0 +1,25 @@
+import { ExpoConfig, ConfigContext } from "@expo/config"
+
+/**
+ * Use ts-node here so we can use TypeScript for our Config Plugins
+ * and not have to compile them to JavaScript
+ */
+require("ts-node/register")
+
+/**
+ * @param config ExpoConfig coming from the static config app.json if it exists
+ * 
+ * You can read more about Expo's Configuration Resolution Rules here:
+ * https://docs.expo.dev/workflow/configuration/#configuration-resolution-rules
+ */
+module.exports = ({ config }: ConfigContext): Partial<ExpoConfig> => {
+  const existingPlugins = config.plugins ?? []
```

Comment:
@mazenchami TS didn't like when I just spread `...plugins` inline since it could be undefined - any other way you like achieving this or does this look fine?

Reply:
this looks fine to me. no need in over complicating it with more checks